deploy/aws/java11Exec/src/main/java/poseidon/App.java

```diff
@@ -95,6 +95,8 @@ public APIGatewayProxyResponseEvent handleRequest(final APIGatewayV2WebSocketEve
 
             ProcessBuilder pb = new ProcessBuilder(cmd);
             pb.directory(workingDirectory);
+            Map<String, String> env = pb.environment();
+            env.put("CLASSPATH", ".:/var/task/lib/org.hamcrest.hamcrest-core-3.0.jar:/var/task/lib/junit.junit-4.13.2.jar:" + env.get("CLASSPATH"));
```

Comment:
I am not sure whether the name of the hamcrest JAR is still correct. Since we made the necessary switch from `hamcrest-core` to `hamcrest`, I would assume that the filename changed here, too. At least the download I can start is located at <https://repo1.maven.org/maven2/org/hamcrest/hamcrest/3.0/hamcrest-3.0.jar>, showing just the `hamcrest-3.0.jar` name.

```suggestion
            env.put("CLASSPATH", ".:/var/task/lib/org.hamcrest.hamcrest-3.0.jar:/var/task/lib/junit.junit-4.13.2.jar:" + env.get("CLASSPATH"));
```